python/eos/analysis_file.py

```diff
@@ -185,9 +184,18 @@ def observables(self, _posterior, _prediction, parameters):
             for key, value in options_part:
                 if key in global_options and global_options[key] != value:
                     eos.error(f'Global option {key}={global_options[key]} overrides option part specification {key}={value} for observable {o.name} in prediction {_prediction}.')
+                if key in o.options and o.options[key] != value:
+                    eos.error(f'Local option {key}={o.options[key]} overrides option part specification {key}={value} for observable {o.name} in prediction {_prediction}.')
```

Comment:
I forgot to ask during the group meeting. Shouldn't one of these errors be a warning instead? @dvandyk?

Reply:
I used `eos.error` as that was what was already being raised for the existing case of options overriding. Although they all would seem to make more sense as warnings

Reply:
This one should probably stay an error, since it might be a bit surprising that an option part specification is overwritten by the local options.